docs/cookbooks/multus/assets/testbed-multus-macvlan.svg

```diff
@@ -0,0 +1,1931 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Do you think better to add "br-int" to the OVS bridge box?

Reply:
Done, thanks for pointing this out, I don't know how I missed adding this the first time